l10n_jp_hr_employee/tests/test_l10n_jp_hr_employee.py

```diff
@@ -18,7 +18,7 @@ def test_00_update_own_private_info_without_self_edit(self):
         self.env['ir.config_parameter'].sudo().set_param(
             'hr.hr_employee_self_edit', False)
         with self.assertRaises(AccessError):
-            self.test_user.with_user(self.test_user.id).write({
+            self.test_user.with_user(self.test_employee.id).write({
```

Comment:
@yostashiro What is the problem of using `test_user`, I think we should pass the user instead of the employee?